clang/test/CodeGen/riscv32-xcorev-hwlp-memcpy.c

```diff
@@ -0,0 +1,75 @@
+// RUN: %clang_cc1 -triple riscv32 -target-feature +xcorevhwlp -S %s -o - \
```

Comment:
Oh, I planned to change that before opening this PR. I just realized, that this test is then pretty useless, since the llvm output is just `call void @llvm.memcpy`, which doesn't check if the HWLP optimization is performed. I'll remove this test.